README.md

```diff
@@ -213,6 +213,12 @@ Tags are fetched in order to correctly generate versioning metadata.
 To view code coverage results open `coverage/index.html` in your browser
 after running `npm run test`.
 
+> If you run `npm run test` and the tests are running but the coverage report is not getting generated
```

Comment:
@michhyun1 Please review the following documentation that I added and provide your feedback. I want you to review this since this might be helpful in improving GitHub actions in the future; I don't think there is a need right now.

The unit test GitHub actions is now working after calling `await device.stop();`

Reply:
This is a good call out - this only occurs when initializing a video processor, right? all the other tests still pass?